appengine/oauth2/pom.xml

```diff
@@ -0,0 +1,60 @@
+<!--
+Copyright 2015 Google Inc. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <packaging>war</packaging>
+  <version>1.0-SNAPSHOT</version>
+  <groupId>com.example.appengine</groupId>
+  <artifactId>appengine-oauth2</artifactId>
+
+  <properties>
+    <appengine.sdk.version>1.9.34</appengine.sdk.version>
```

Comment:
[nit, optional] shouldn't need to specify appengine.sdk.version, since the parent also defines such a property. We can keep it here if we want pom.xml to work (mostly) without the parent present.